src/components/MainPage.js

```diff
@@ -0,0 +1,105 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+// import car from '../img/lambo.webp';
```

Comment:
- [ ] Please, kindly delete this old piece of commented code, to improve the cleanliness of the app. 💪